src/app/app.component.ts

```diff
@@ -27,7 +27,9 @@ export class AppComponent implements OnInit {
   constructor(
     public store: Store<any>,
     private angulartics2GoogleAnalytics: Angulartics2GoogleAnalytics
-  ) {}
+  ) {
```

Comment:
Publish is a version behind metrics on the Google Analytics library. This is the new way to initialize it.

Reply:
+1